pyfreenas/pool.py

```diff
@@ -0,0 +1,146 @@
+from enum import Enum, unique
+from typing import TypeVar
+
+TMachine = TypeVar("TMachine", bound="Machine")
+TPoolStatus = TypeVar("TType", bound="PoolStatus")
+# TPoolScanState = TypeVar("TType", bound="PoolScanState")
+# TPoolEncryptionAlgorithm = TypeVar("TType", bound="PoolEncryptionAlgorithm")
```

Comment:
Can you remove these instead of commenting them out?

Reply:
Removed.